Spigot-API-Patches/0001-POM-changes.patch

```diff
@@ -110,6 +110,19 @@ index 8964b9e33..ce09baeab 100644
                      <!-- when downloading via Maven we can pull depends individually -->
                      <shadedArtifactAttached>true</shadedArtifactAttached>
                  </configuration>
+diff --git a/src/main/java/org/bukkit/event/player/PlayerEvent.java b/src/main/java/org/bukkit/event/player/PlayerEvent.java
+index 793b661b..b7c8f2c3 100644
+--- a/src/main/java/org/bukkit/event/player/PlayerEvent.java
++++ b/src/main/java/org/bukkit/event/player/PlayerEvent.java
+@@ -14,7 +14,7 @@ public abstract class PlayerEvent extends Event {
+         player = who;
+     }
+ 
+-    PlayerEvent(@NotNull final Player who, boolean async) {
++    public PlayerEvent(@NotNull final Player who, boolean async) { // Paper - wtf?
```

Comment:
I do not think this change is in the correct patch?

Reply:
I didn't want to make a new patch for it lol

Reply:
though i was thinking we already had other crap in pom patch, but thats on -Server